src/Mod/Start/Gui/FirstStartWidget.cpp

```diff
@@ -98,7 +91,7 @@ bool FirstStartWidget::eventFilter(QObject* object, QEvent* event)
 
 void FirstStartWidget::retranslateUi()
 {
-    _doneButton->setText(tr("Done"));
+    doneButton->setText(tr("Done"));

```

Comment:
Is there a way to hack this so it doesn't touch translation code ?

Reply:
The underscore is just a naming convention for private variables afaik, I removed it since that button needed to be public for signal handling. We can just keep the underscore if the variable name messes with translation.

Reply:
Just tested and seems like it doesn't affect translation anyway.